src/healthRecord/dto/healthRecord.dto.ts

```diff
@@ -0,0 +1,30 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsArray, IsDate, IsObject, IsString } from 'class-validator';
+
+export class HealthDataDto {
+  @ApiProperty()
+  @IsDate()
+  dateTime: Date;
+
+  @ApiProperty()
+  @IsArray()
+  values: Array<string>;
+}
+
+export class HealthRecordDto {
```

Comment:
Would it be possible to use a different name for this Dto? It's conflicting with mine currently

Reply:
Can't we share the same name or is it extensible? What are attributes used in your dto?

Reply:
If it is really not possible to extend from the same dto, I am considering changing it to healthRecordTable

Reply:
hrName and imageid

Reply:
ive changed mine to healthRedordtable